materialrangebar/src/main/res/values/attrs.xml

```diff
@@ -24,6 +25,9 @@
         <attr name="mrb_pinMaxFont" format="dimension"/>
         <attr name="mrb_rangeBarPaddingBottom" format="dimension"/>
         <attr name="mrb_selectorColor" format="reference|color"/>
+        <attr name="mrb_selectorColorLeft" format="reference|color"/>
+        <attr name="mrb_selectorColorRight" format="reference|color"/>
+        <attr name="mrb_selectorColorChange" format="boolean"/>
```

Comment:
This is not being used so can be removed

Reply:
Just remove mrb_selectorColorChange right?

Reply:
yup